deprecate.dd

```diff
@@ -116,56 +115,6 @@ $(H4 Rationale)
     $(P Hexstrings are used so seldom that they don't warrant a language feature.
     )
 
-$(H3 $(DEPNAME Class allocators and deallocators))
-    $(P D classes can have members customizing the (de)allocation strategy.
-        ---
-        class Foo
-        {
-            new(uint size, ...)
-            {
-                return malloc(size);
-            }
-
-            delete(void* obj)
-            {
-                free(obj);
-            }
-        }
-
-        Foo foo = new(...) Foo();
-        delete foo;
-        ---
-    )
-$(H4 Corrective Action)
-    $(P Move the (de)allocation strategy out of the class
-        ---
-        class Foo
-        {
-        }
-
-        T make(T, Args...)(auto ref Args args) if (is(T == Foo))
-        {
-            enum size = __traits(classInstanceSize, T);
-            void* mem = malloc(size);
-            scope (failure) free(mem);
-            return mem !is null ? emplace!T(mem[0..size], args) : null;
-        }
-
-        void dispose(T)(T obj)
-        {
-            auto mem = cast(void*) obj;
-            scope (exit) free(mem);
-            destroy(obj);
-        }
-
-        Foo foo = make!Foo();
-        if (foo !is null) dispose(foo);
-        ---
-    )
-$(H4 Rationale)
-    $(P Classes should not be responsible for their own (de)allocation strategy.
-    )
```

Comment:
No this needs to be kept. The entire point of this document is to provide historic rationals.